packages/tools/pluggable-widgets-tools/configs/webpack.native.config.js

```diff
@@ -48,7 +53,7 @@ const widgetConfig = {
                     loader: "babel-loader",
                     options: {
                         cacheDirectory: true,
-                        presets: ["@babel/preset-env", "@babel/preset-react"],
+                        presets: ["module:metro-react-native-babel-preset", "@babel/preset-react"],
```

Comment:
Are you sure this change is necessary? If yes we need to send this task to test to see if there is not real changes in the widget while in SP

Reply:
On the metro bundler side this preset is also used. The thing is however that webpack 4 doesn't support optional chaining syntax output from loaders. This will start to be supported from Webpack 5. That's why I decided to bundle native widgets as the desired format for metro bundler / React Native.

Reply:
Ok, im fine with the change, we just need to build all native widgets in Mono repo and see if they behave the same with this

Reply:
- But why is it necessary? Metro bundler will process this javascript regardless of our wishes (as part of bundling). Hence it will anyway run `metro-react-native-babel-preset`. Hence (if the reasoning is correct) we don't have to do it ourselves.
- If we decide that we should run `metro-react-native-babel-preset`, then custom webpack configs above are not needed, since `@babel/plugin-transform-flow-strip-types` is part of metro preset.

Reply:
@mijay 

1) Babel-loader will output optional chaining, which Webpack cannot handle. So we need to process js, so that it doesn't contain optional chaining anymore. That's why a chose to go for the desired end output of RN / Metro bundler.

2) Good point, I will test this.

Reply:
This issue has been [recently fixed](https://github.com/webpack/webpack/pull/11221) in a webpack

Reply:
As discussed with @mijay I kept `metro-react-native-babel-preset` in the config. I removed the local configs for `toggle-buttons-native` as this is no longer necessary due to the metro preset.